src/shared/components/Stepper/Stepper.tsx

```diff
@@ -1,47 +1,50 @@
 import React, { useEffect, useState } from 'react'
 
-import { SvgGlyphFileVideo, SvgGlyphLock, SvgGlyphTrash } from '@/shared/icons'
-import { FileType } from '@/types/files'
+import { SvgGlyphCheck, SvgGlyphFileVideo, SvgGlyphLock, SvgGlyphTrash } from '@/shared/icons'
 
 import {
-  FileName,
   Overhead,
   StepDetails,
+  StepImage,
   StepNumber,
   StepStatus,
+  StepTitle,
   StepWrapper,
   StyledProgress,
-  Thumbnail,
-} from './FileStep.style'
+} from './Stepper.styles'
 
 import { IconButton } from '../IconButton'
 
-export type FileStepProps = {
-  stepNumber: number
-  active: boolean
-  onDelete: (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void
-  type: FileType
-  isFileSet?: boolean
+export type StepperProps = {
+  title: string
+  variant?: 'file' | 'default'
+  completed?: boolean
   thumbnailUrl?: string | null
-  onSelect?: (step: FileType) => void
-  disabled?: boolean
   isLoading?: boolean
+  type?: 'video' | 'image'
+  isFileSet?: boolean
+  disabled?: boolean
+  active?: boolean
+  number?: number
+  onClick?: () => void
+  onDelete?: () => void
 }
-
-export const FileStep: React.FC<FileStepProps> = ({
-  stepNumber = 1,
-  active,
-  isFileSet,
-  type,
-  onDelete,
+export const Stepper: React.FC<StepperProps> = ({
```

Comment:
I think this should be named `Step`. `Stepper` suggests it's a collection of steps and this is only a single one